source/css/scss-common/base/_forms.scss

```diff
@@ -526,197 +539,134 @@ textarea {
-      }
-    }
-  }
+.a-custom-radio+.a-custom-radio {
+  margin-left: 0;
 }
 
 // RADIOBUTTONS WITH EXCERPT
 .a-radioButtons-stackedList {
-  .custom-control+.custom-control {
-    margin-left: 0;
-  }
-
-  .a-legend {
-    @include a-fontSize24;
-    color: $black;
-  }
+  display: block;
 
-  .custom-radio {
+  .a-radioButtons-excerpt {
```

Comment:
Should we also change the name 'custom-radio' in line 674 to 'a-radiobuttons-excerpt'?